services/QuillLMS/client/app/bundles/Connect/components/activityHealth/activityHealth.tsx

```diff
@@ -0,0 +1,417 @@
+              }) : ''
+            }
+          </div>
+        ),
+        minWidth: 180
+      },
+      {
+        Header: "Plays in Last 3 Months",
+        accessor: 'recent_plays',
+        filterMethod: filterNumbers,
+        Filter: ({ filter, onChange }) =>
+          (
+            <NumberFilterInput
+              filter={filter}
+              handleChange={onChange}
+              label="Filter for recent plays"
+            />
+          ),
+        resizeable: true,
+        sortMethod: sort,
+        Cell: props => this.addCommasToThousands(props.value),
+        maxWidth: 90
+      },
+      {
+        Header: 'Activity Packs',
+        accessor: 'activity_packs',
+        filterMethod: (filter, rows) =>
+              matchSorter(rows, filter.value, { keys: ["activity_packs.*.name"] }),
+        filterAll: true,
+        resizeable: true,
+        sortMethod: sortByList,
+        Cell: (row) => (
+          <div>
+            {
+              row.original['activity_packs'] ?
+              row.original['activity_packs'].map((ap, index) => {
+                if (!ap.name) return "";
+                else if (index != row.original['activity_packs'].length - 1) return <div key={ap.id}>{ap.name},</div>
+                else return <div key={ap.id}>{ap.name}</div>
+              }) : ''
+            }
+          </div>
+        ),
+        minWidth: 180
+      },
+      {
+        Header: 'Average Difficulty',
+        accessor: 'avg_difficulty',
+        filterMethod: filterNumbers,
+        Filter: ({ filter, onChange }) =>
+        (
+          <NumberFilterInput
+            filter={filter}
+            handleChange={onChange}
+            label="Filter for average difficulty"
+          />
+        ),
+        resizeable: true,
+        sortMethod: sort,
+        Cell: props => props.value,
+        maxWidth: 150
+      },
+      {
+        Header: 'Standard Deviation Difficulty',
+        accessor: 'standard_dev_difficulty',
+        filterMethod: filterNumbers,
+        Filter: ({ filter, onChange }) =>
+        (
+          <NumberFilterInput
+            filter={filter}
+            handleChange={onChange}
+            label="Filter for recent plays"
+          />
+        ),
+        resizeable: true,
+        sortMethod: sort,
+        Cell: props => props.value,
+        maxWidth: 150
+      },
+      {
+        Header: 'Average Common Unmatched',
+        accessor: 'avg_common_unmatched',
+        filterMethod: filterNumbers,
+        Filter: ({ filter, onChange }) =>
+        (
+          <NumberFilterInput
+            filter={filter}
+            handleChange={onChange}
+            label="Filter for avg common unmatched"
+          />
+        ),
+        resizeable: true,
+        sortMethod: sort,
+        Cell: props => props.value,
+        maxWidth: 150
+      }
+    ];
+  }
+
+  addCommasToThousands = (num) =>
+  {
+    if (!num) return ""
+    var num_parts = num.toString().split(".");
```

Comment:
Let's use `const` or `let` instead of var: https://github.com/airbnb/javascript#references